gulpfile.js

```diff
@@ -64,18 +57,6 @@ gulp.task('clean', function cleanTask() {
   ]);
 });
 
-gulp.task('npm:tabzilla:img', function npmTabzillaImgTask() {
-  return gulp.src(NODE_MODULES_PATH + 'mozilla-tabzilla/media/**')
-    // the tabzilla css looks for images in "../media/img/" :-(
-    .pipe(gulp.dest(DEST_PATH + 'media/'));
-});
-
-gulp.task('vendor', function vendorTask(done) {
-  return runSequence([
-    'npm:tabzilla:img'
-  ], done);
-});
```

Comment:
You'll need to remove the `vendor` task reference from `gulp.task('build')` on L141 below as well, since it's blowing up Travis-CI:

```
[22:23:14] Using gulpfile /home/travis/build/mozilla/testpilot/gulpfile.js
[22:23:14] Starting 'build'...
[22:23:14] 'build' errored after 1.09 ms
[22:23:14] Error: Task vendor is not configured as a task on gulp.  If this is a submodule, you may need to use require('run-sequence').use(gulp).
    at /home/travis/build/mozilla/testpilot/node_modules/run-sequence/index.js:20:10
    at Array.forEach (native)
    at verifyTaskSets (/home/travis/build/mozilla/testpilot/node_modules/run-sequence/index.js:13:11)
    at runSequence (/home/travis/build/mozilla/testpilot/node_modules/run-sequence/index.js:94:2)
    at Gulp.buildTask (/home/travis/build/mozilla/testpilot/gulpfile.js:139:3)
    at module.exports (/home/travis/build/mozilla/testpilot/node_modules/gulp/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/home/travis/build/mozilla/testpilot/node_modules/gulp/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/home/travis/build/mozilla/testpilot/node_modules/gulp/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/home/travis/build/mozilla/testpilot/node_modules/gulp/node_modules/orchestrator/index.js:134:8)
    at /home/travis/build/mozilla/testpilot/node_modules/gulp/bin/gulp.js:129:20
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
```